libbeat/tests/system/beat/beat.py

```diff
@@ -29,6 +31,17 @@ def start(self):
             stderr=subprocess.STDOUT)
         return self.proc
 
+    def kill(self):
+        if sys.platform.startswith("win"):
```

Comment:
You can verify #559 is fixed by looking at the zip file containing the test outputs on AppVeyor. If there are a bunch of `.cov` files, then it's fixed. :smiley_cat: